### DISCUSSION TITLE
Different results with same settings from 2.3.1

### DISCUSSION THREAD

**Discussion by Krashlog** [lllyasviel/Fooocus#3007] [Q&A] [answered]
Hello,

I'm having a hard time getting good results for 2.4.0 with prompts that used to consistently give me good output. I've taken a look at previous gens created with 2.3.1 and tried using the same metadata found in the logs (same seed, prompt, model, etc) and compare the results. Now I don't want to call them "worse" because that's in the eye of the beholder, but they definitely don't give me the same output.

Would the fact that the prompt is slightly "NSFW" explain that difference? Or is it something else?

Thanks!
PS: I hesitate to share the results and prompts directly here since they are NSFW, but if that could help I can send the metadata for repro.

**Comment by ILikeToasters** [comment, marked as answer]
I did already post this in the update discussion but will here as well. I would suggest further discussion is done there so it may speed up figuring out any issues. I can confirm that for some images they are not recreated, but for some I can recreate the exact image. So it may be something specific to certain models.

**Reply by Krashlog** [reply]
Very well, let's discuss this over there then!

**Comment by mashb1t** [comment]
=> see https://github.com/lllyasviel/Fooocus/discussions/3003#discussioncomment-9563985
